Main.java

```diff
@@ -0,0 +1,15 @@
+public class Main{
+    public static void main(String[] args){
+        if(args.length>0){
+        for(String val: args)
+        System.out.println(val);
+    }
+        else{
+
+            System.out.println("There is no arguments are provided");
+
+        
+    }
```

Comment:
Please only commit formatted code!

Reply:
yeah! we understood what you said. I will push same !!